app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -32,5 +33,16 @@ def destroy
       form.destroy
       render json: form
     end
+
+    def proxy_add
+      if params[:id] == '21-526EZ'
```

Comment:
Could we use the constant `VA526ez::FORM_ID` rather than the string `21-526EZ`?

Reply:
Yes, thats a good idea.

Reply:
Absolutely-- I didn't realize it was set as a constant somewhere. Thanks!